common/sgx/collateral.c

```diff
@@ -247,9 +247,6 @@ oe_result_t oe_validate_revocation_list(
             version,
             OE_SGX_ENDORSEMENTS_VERSION);
 
-    OE_STATIC_ASSERT(
-        OE_COUNTOF(crl_issuer_chain) >= OE_SGX_ENDORSEMENTS_CRL_COUNT);
```

Comment:
This assertion should stay and check 'crls' instead because OE_SGX_ENDORSEMENTS_CRL_COUNT is the max index for writing to crls[i].

Reply:
Updated.